source/draw/DrawState.ooc

```diff
@@ -29,6 +29,8 @@ DrawState: cover {
 	opacity := 1.0f
 	_transformNormalized := FloatTransform3D identity
 	viewport := IntBox2D new(0, 0, 0, 0)
+	destination := IntBox2D new(0, 0, 0, 0) // TODO: Accept FloatBox2D
```

Comment:
This version is a wrapper on the old API that only takes integers. The next step is much more complex and might break backward compability if I create a bug.